radicle-cli/examples/rad-patch.md

```diff
@@ -29,7 +29,7 @@ $ rad patch open --message "Define power requirements" --message "See details."
 ✓ Pushing HEAD to storage...
 ✓ Analyzing remotes...
 
-z6MknSL…StBU8Vi/master (f2de534) <- z6MknSL…StBU8Vi/flux-capacitor-power (3e674d1)
```

Comment:
The ref on the right is important, it's the head of the branch you want to merge.